docker/base/os/ubuntu/Dockerfile.jammy

```diff
@@ -0,0 +1,20 @@
+# This is the dockerfile for generating base images for ubuntu Jammy Jellyfish
+# contains the minimal dependencies needed for V to run
+FROM ubuntu:22.04
```

Comment:
If you want to keep up with the latest, just remove the version number.  Whenever the image is built, it will use `ubuntu:latest`.

Reply:
ok, I confirm that `ubuntu:latest` is a reference to `ubuntu:jammy` (so latest LTS); but this is a simple way to ensure this image will always use this major version.
What do you think if I'd add a `Dockerfile.latest` and always use it in build pipelines ?